index.html

```diff
@@ -60,6 +71,242 @@ <h2 class="subtitle">I'm a Software Developer</h2>
           </ul>
         </article>
       </section>
+
+      <section class="works">
+        <header class="works_title">
+          <h2>My Recent Works</h2>
+        </header>
+        <article class="works_content">
+          <hr />
+          <img src="icons/Img_place.png" alt="The Banner" />
+          <h2>Multi-Post Stories</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul>
+            <li>
+              <a href="#">CSS</a>
+            </li>
+            <li>
+              <a href="#">HTML</a>
+            </li>
+            <li>
+              <a href="#">BOOTSTRAP</a>
+            </li>
+            <li>
+              <a href="#">RUBY</a>
+            </li>
+          </ul>
+          <button type="button" >See project</button>
+        </article>
+      </section>
+
+      <section class="center_content">
+        <article class="content_desc">
+          <h2>Profesional Art <br />Printing Data</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul>
+            <li>
+              <a href="#">HTML</a>
+            </li>
+            <li>
+              <a href="#">BOOTSTRAP</a>
+            </li>
+            <li>
+              <a href="#">RUBY</a>
+            </li>
+          </ul>
+          <button type="button" >See project</button>
+        </article>
```

Comment:
- [ ] Please use all button effects I notice your pressed effect is not working so kindly fix this issue :+1:  check below :point_down: the pressed button effect :slightly_smiling_face: 

![Screenshot from 2023-04-29 14-53-54](https://user-images.githubusercontent.com/45226848/235296902-4a839471-8d0a-4f53-96eb-f5836b5fde13.png)